src/com/xilinx/rapidwright/util/RelocateModulesIntoBlackboxes.java

```diff
@@ -98,13 +102,27 @@ public static boolean relocateModuleInsts(Design top, Module mod, String cellAnc
 			}
 
 			ModuleInst mi = top.createModuleInst(cell.getFirst(), mod, true);
-			mi.getCellInst().setCellType(mod.getNetlist().getTopCell());
 			mi.place(toSite);
 		}
 		System.out.println("\n");
 		return true;
 	}
 
+	/**
+	 * Process the design after filing black boxes.
+	 * @param top The design to process
+	 * @param blackboxes The black boxes that was filled
+	 */
+	public static void postProcessing(Design top, List<Pair<String, String>> blackboxes) {
+
+		top.getNetlist().resetParentNetMap();
+
+		for (Pair<String, String> toCellLoc : blackboxes) {
+			combinePIPonClockNets(top, toCellLoc.getFirst());
```

Comment:
I know this is just a refactoring, but in trying to understand `combinePIPonClockNets()` it wasn't clear whether this method is similar to https://github.com/Xilinx/RapidWright/blob/0b5183ca63ea7669e1605b08d4807ac061dbe942/src/com/xilinx/rapidwright/design/DesignTools.java#L2721
?

Reply:
They are different in that combinePIPonClockNets just move PIP between subnets (ie., ) to match what Vivado expects, without merging or removing nets like makePhysNetNameConsistent (, which I don't understand exactly the intend of the method)

Reply:
These methods are similar.  The `makePhysNetNamesConsistent()` goes through the entire design, and replaces any net names with the correct parent net name.  It doesn't necessarily fix/combine routing.